script/git-clone-others.sh

```diff
@@ -57,6 +57,8 @@ additionalGitOptions=()
 REPOSITORY_LIST=`cat "${scriptDir}/repository-list.txt"`
 # Repositories that need to use the branch 7.x instead of master
 BRANCHED_7_REPOSITORY_LIST=`cat "${scriptDir}/branched-7-repository-list.txt"`
+# Repositories thta ned to use branch main instead of master
+REPOSITORY_MAIN_LIST=`cat "${scriptDir}/repository-main-list.txt"`
```

Comment:
I would suggest renaming to avoid confusion with the meaning of a "main list of repositories". Something such as `MAIN_BRANCH_REPOSITORY_LIST` or `REPOSITORIES_WITH_MAIN_BRANCH`?

Reply:
renamed. DONE